live-build/config/hooks/configuration/download-packages.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -eu
+
+set -o pipefail
+
+function download_packages() {
+	echo "Downloading: $*"
+	for _ in {1..5}; do
+		apt-get download "$@" && return
+		echo "Download failed, retrying."
+		sleep 5
+	done
+	echo "Failed to download packages after 5 attempts. Aborting." >&2
+	exit 1
```

Comment:
we use `exit` here, but `return` earlier in this function.. we should try to be consistent and either use `exit` for both, or `return` for both.